functions/test/endpoints/FirestoreDataFactory.ts

```diff
@@ -168,28 +168,33 @@ export class FirestoreDataFactory{
      * @param db - Test App Firestore instance (Usually from authedApp())
      * @param house - name of the house to add point log for
      * @param resident_id - id of resident to add point log for
-     * @param approved - boolean for if the pointlog was already approved. (used to set the approved fields and sign of point type ID)
+     * @param handled - boolean for if the pointlog was already handled. (used to set the approved fields and sign of point type ID)
      * @param ptOpts - Optional parameters to modifu the point log
      */
-    static setPointLog(db: firebase.firestore.Firestore, house:string, resident_id:string, approved: boolean, ptOpts:PointLogOptions = POINT_LOG_DEFAULTS): Promise<void> | Promise<firebase.firestore.DocumentReference>{
+    static setPointLog(db: firebase.firestore.Firestore, house:string, resident_id:string, handled: boolean, ptOpts:Options.PointLogOptions = Options.POINT_LOG_DEFAULTS): Promise<void> | Promise<firebase.firestore.DocumentReference>{
```

Comment:
Clarified the setPointLog method to take a parameter 'handled' instead of 'approved'.

Also added an optional 'approved' field to the POINT_LOG_DEFAULTS